httpx/_multipart.py

```diff
@@ -200,7 +199,7 @@ def __init__(
         boundary: typing.Optional[bytes] = None,
     ) -> None:
         if boundary is None:
-            boundary = binascii.hexlify(os.urandom(16))
+            boundary = ''.join([f'{byte:02x}' for byte in(os.urandom(16)])
```

Comment:
Oh, I see python 2 user 😄 
```suggestion
            boundary = os.urandom(16).hex()
```